features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelineView.kt

```diff
@@ -232,6 +256,36 @@ fun TimelineItemEventRow(
     }
 }
 
+@Composable
+private fun TimestampView(
+    sentTime: String,
+    isEdited: Boolean,
+    sendingFailed: Boolean,
+    onResendClicked: () -> Unit,
```

Comment:
Maybe some wording discussion, but at this point, `TimestampView` cannot know the action that will be done once it is clicked. I would rename to `onTimestampClicked`, or just `onClicked`.
In the same spirit, `sendingFailed` could be rename to `isError` (or maybe something better)

Reply:
Changed them in https://github.com/vector-im/element-x-android/pull/465/commits/a37a5523ceab1494f2bfefef4d64e191c2acc59f, it should be better now.